### PR TITLE
table/iterator.cc: fix typos in comments

### PR DESCRIPTION
**Pull request by rockeet**

### CODE REVIEW

File: table/iterator.cc [hunk 1]
```diff
@@ -35,13 +35,13 @@ Cleanable& Cleanable::operator=(Cleanable&& other) {
...
 // If the entire linked list was on heap we could have simply add attach one
-// link list to another. However the head is an embeded object to avoid the cost
+// link list to another. However the head is an embedded object to avoid the cost
 // of creating objects for most of the use cases when the Cleanable has only one
 // Cleanup to do. We could put evernything on heap if benchmarks show no
 // negative impact on performance.
 // Also we need to iterate on the linked list since there is no pointer to the
-// tail. We can add the tail pointer but maintainin it might negatively impact
-// the perforamnce for the common case of one cleanup where tail pointer is not
+// tail. We can add the tail pointer but maintain it might negatively impact
```
Comment (mrambacher) [inline table/iterator.cc:43]: Should be "maintaining"

File: table/iterator.cc [hunk 2]
```diff
@@ -35,13 +35,13 @@ Cleanable& Cleanable::operator=(Cleanable&& other) {
 }
 
 // If the entire linked list was on heap we could have simply add attach one
-// link list to another. However the head is an embeded object to avoid the cost
+// link list to another. However the head is an embedded object to avoid the cost
 // of creating objects for most of the use cases when the Cleanable has only one
 // Cleanup to do. We could put evernything on heap if benchmarks show no
```
Comment (mrambacher) [inline table/iterator.cc:40]: "everything"

**Comment by jay-zhuang** [comment]
@rockeet would you please update based on Mark's comments and run `make format`?

**Comment by jay-zhuang** [comment]
As we don't have permission to update the contributor's branch, closing for now. @rockeet please reopen after `make format` and other minor fixes.

**Pull request closed without merging**